Examples/Fermions/fermi_hubbard.py

```diff
@@ -54,7 +54,7 @@ def nc(site, sz):
 
 # g.n_nodes == L*L --> disj_graph == 2*L*L
 disj_graph = nk.graph.disjoint_union(g, g)
-sa = nk.sampler.MetropolisExchange(hi, graph=disj_graph, n_chains=16)
+sa = nkx.sampler.MetropolisFermionExchange(hi, graph=disj_graph, n_chains=16)
```

Comment:
since we are doing a specific sampler for fermions (which is very much needed, thanks!) why don't we make it such we don't need to do anymore this hack of the disjoint union?